index.html

```diff
@@ -58,15 +58,16 @@
             <dt>-b:a <i>320k</i></dt><dd>This sets the bit rate at the highest rate the MP3 format allows. Reduce this to 160k for mono files.</dd>
             <dt><i>output_file</i></dt><dd>path and name of the output file</dd>
           </dl>
+          <p class="link"></p>
         </div>
       </div>
     </div>
   </div>
   <!-- ends WAV to MP3 -->
 
   <!-- Transcode to ProRes -->
-  <span data-toggle="modal" data-target=".to_prores"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This will transcode to deinterlaced Apple ProRes LT">Transcode to ProRes</button></span>
-  <div class="modal fade to_prores" tabindex="-1" role="dialog">
+  <span data-toggle="modal" data-target="#to_prores"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="This will transcode to deinterlaced Apple ProRes LT">Transcode to ProRes</button></span>
+  <div id="to_prores" class="modal fade" tabindex="-1" role="dialog">
```

Comment:
The _Transcode to ProRes_ section does **not** contain the
`<p class="link"></p>`
part. Yet it seems to me to work fine. Is only one occurrence needed?